src/web/types/question-details-impl/feedback-rank-recipients-question-details.impl.ts

```diff
@@ -45,19 +46,24 @@ export class FeedbackRankRecipientsQuestionDetailsImpl extends AbstractFeedbackQ
       'Self Rank',
       'Overall Rank',
       'Overall Rank Excluding Self',
-      'Ranks Received',
       'Team Rank',
       'Team Rank Excluding Self',
+      'Ranks Received',
     ]);
 
     Object.keys(statsCalculation.ranksReceivedPerOption).sort().forEach((recipient: string) => {
       statsRows.push([
         statsCalculation.emailToTeamName[recipient],
         statsCalculation.emailToName[recipient],
-        statsCalculation.selfRankPerOption[recipient] ? String(statsCalculation.selfRankPerOption[recipient]) : '-',
-        statsCalculation.rankPerOption[recipient] ? String(statsCalculation.rankPerOption[recipient]) : '-',
+        statsCalculation.selfRankPerOption[recipient] ? String(statsCalculation.selfRankPerOption[recipient])
+        : emptyStr,
```

Comment:
Maybe just line break at `?` instead of `:`, like line 62-63

Reply:
Yes, it looks cleaner with your suggestion